client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/config/processor/ZeebeWorkerSpringJobHandler.java

```diff
@@ -0,0 +1,118 @@
+package io.camunda.zeebe.spring.client.config.processor;
+
+import io.camunda.zeebe.client.api.command.CompleteJobCommandStep1;
+import io.camunda.zeebe.client.api.response.ActivatedJob;
+import io.camunda.zeebe.client.api.worker.JobClient;
+import io.camunda.zeebe.client.api.worker.JobHandler;
+import io.camunda.zeebe.client.impl.Loggers;
+import io.camunda.zeebe.spring.client.annotation.ZeebeVariable;
+import io.camunda.zeebe.spring.client.bean.ParameterInfo;
+import io.camunda.zeebe.spring.client.bean.value.ZeebeWorkerValue;
+import io.camunda.zeebe.spring.client.exception.ZeebeBpmnError;
+import org.slf4j.Logger;
+
+import java.io.InputStream;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ZeebeWorkerSpringJobHandler implements JobHandler {
+
+  private static final Logger LOG = Loggers.JOB_WORKER_LOGGER;
+  private ZeebeWorkerValue workerValue;
+
+  public ZeebeWorkerSpringJobHandler(ZeebeWorkerValue workerValue) {
+    this.workerValue = workerValue;
+  }
+
+  @Override
+  public void handle(JobClient jobClient, ActivatedJob job) throws Exception {
```

Comment:
I have the feeling that a lot of this figuring out the parameter classes and result class can be done once, and should be done on every new job. But I think it's something we can improve afterwards and doesn't have to block this improvement.

Reply:
I agree with both comments :-)

Reply:
I add a comment to the existing code